git-cliff/src/lib.rs

```diff
@@ -160,18 +160,20 @@ fn process_repository<'a>(
 
 	// Parse commits.
 	let mut commit_range = args.range.clone();
+	let mut include_root = false;
 	if args.unreleased {
 		if let Some(last_tag) = tags.last().map(|(k, _)| k) {
 			commit_range = Some(format!("{last_tag}..HEAD"));
 		}
 	} else if args.latest || args.current {
 		if tags.len() < 2 {
-			let commits = repository.commits(None, None, None)?;
+			let commits = repository.commits(None, false, None, None)?;
 			if let (Some(tag1), Some(tag2)) = (
 				commits.last().map(|c| c.id().to_string()),
 				tags.get_index(0).map(|(k, _)| k),
 			) {
 				commit_range = Some(format!("{tag1}..{tag2}"));
```

Comment:
No need for an additional flag here, just doing this would be enough:

```rust
if tags.len() == 1 {
	commit_range = Some(format!("{tag2}"));
} else {
	commit_range = Some(format!("{tag1}..{tag2}"));
}
```